src/fortune-wheel-booth-frontend/fortune-wheel/FortuneWheelPage.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import { useCallback, useEffect, useState } from 'react';
+import FortuneWheel from '../src/components/FortuneWheel';
+import ModalPrize from '../src/components/ModalPrize';
+import { findPrize } from '../src/utils/findPrize';
+import { Principal } from '@dfinity/principal';
+import { type ActorSubclass } from '@dfinity/agent';
+import {
+  _SERVICE,
+  Extraction,
+} from '../../declarations/fortune-wheel-booth-backend/fortune-wheel-booth-backend.did';
+import {
+  canisterId,
+  createActor,
+} from '../../declarations/fortune-wheel-booth-backend';
+
+function FortuneWheelPage() {
+  const [showModalPrize, setShowModalPrize] = useState(false);
+  const [lastExtraction, setLastExtraction] =
+    useState<[Principal, Extraction]>();
+  const [mustSpin, setMustSpin] = useState(false);
+  const [actor, setActor] = useState<ActorSubclass<_SERVICE>>();
+  const [prizeNumber, setPrizeNumber] = useState<number>(0);
+
+  useEffect(() => {
+    if (showModalPrize)
+      setTimeout(() => {
+        setShowModalPrize(false);
+        setMustSpin(false);
+      }, 8000);
+  }, [showModalPrize]);
+
+  useEffect(() => {
+    const actor = createActor(canisterId);
+    setActor(actor);
+  }, []);
```

Comment:
Why do we need this and can't just set it in a variable in a shared file / at the top of this file?